example/package.json

```diff
@@ -14,16 +14,18 @@
     "express": "^4.12.3",
     "express-partial-templates": "^0.1.0",
     "express-session": "^1.13.0",
-    "hmpo-form-controller": "0.0.6",
-    "hmpo-form-wizard": "0.2.10",
-    "hmpo-frontend-toolkit": "^1.0.3",
+    "hmpo-form-controller": "^0.7.0",
+    "hmpo-form-wizard": "file:..",
```

Comment:
generated by:
npm install --save hmpo-form-wizzard@../
The example installs the for wizzard module this is in, instead of a set version (which would be out of date when this is published)
could do @latest or change all requires to relative paths instead.